ikenna-portfolio/portfolio/src/main/webapp/test.html

```diff
@@ -0,0 +1,267 @@
+    }
+
+    function changeFrontLever() {
+      document.getElementById("cal-image1").src = "/images/front_lever.jpg";
+      document.getElementById("cal-image2").src = "/images/single_leg_front_lever.png";
+      document.getElementById("goal").innerText = goal.replace("-", "Front Lever");
+      document.getElementById("current").innerText = current.replace("-", "Single Leg Front Lever");
+    }
+
+    function changeHSPU() {
+      document.getElementById("cal-image1").src = "/images/hspu.jpg";
+      document.getElementById("cal-image2").src = "/images/decline_pike_push_up.jpg";
+      document.getElementById("goal").innerText = goal.replace("-", "Freestanding Handstand Push Up");
+      document.getElementById("current").innerText = current.replace("-", "Decline Pike Push Up");
+    }
+
+  </script>
+</head>
+<body onload="load()">
+  <div class="theme">
+
+    <!-- Navbar -->
+    <nav class="navbar navbar-default">
+      <div class="container">
+        <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>                         
+        </button>
+        <a class="navbar-brand" href="#">'kenna</a>
+        </div>
+        <div class="collapse navbar-collapse" id="myNavbar">
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#">INTRO</a></li>
+            <li><a href="#">STEP</a></li>
+            <li><a href="#">PROJECTS</a></li>
+            <li><a href="#">CALISTHENICS</a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- First Container -->
+    <div class="container-fluid bg-1 text-center">
+      <h3 class="margin">Ikenna's Portfolio</h3>
+      <img class="image-format" src="/images/img_baby.jpeg" alt="Youngin 'kenna">
+      <h3>I'm a Googler!!</h3>
+      <p class="desc">My name is <b>Ikenna Jonathan Elue</b>. I am a male identifying Nigerian American 
+      from Chicago who currently attends Northwestern University. In the past, I've worked with Chicago 
+      based industries such as JCC Apachi Camp (Jewish Community Center) and ASM (After School Matters). 
+      My favorite food is Nigerian jollof rice. I enjoy doing calisthenic based exercises, combination 
+      jump roping, and dancing. One non physical activity I enjoy is watching comedy animes or cartoons.
+      </p>
+      <br>
+      <p class="desc">Here's a link to my LinkedIn <a href="https://www.linkedin.com/in/ikenna-elue-a28798166/">page</a></p>
+      <p class="desc">Here's a link to my GitHub <a href="https://github.com/ike-elue">page</a></p>
+    </div>
+
+    <!-- Second Container -->
+    <div class="container-fluid bg-2 text-center">
+      <h3 class="margin">STEP So Far</h3>
+      <h3><u>TODO</u></h3>
```

Comment:
Remove?

Reply:
done-ish, improvised because I want to complete the portion and signal to myself in the site to do it soon